package.json

```diff
@@ -64,6 +64,7 @@
     "redux": "4.0.1",
     "redux-logger": "3.0.6",
     "redux-thunk": "2.3.0",
+    "semver": "^7.3.5",
```

Comment:
What is the size added by this lib to the bundle? 

There is no bundlemoon on this repo yet

Reply:
The bundle went from 3552 Bytes to 3580 :)